src/components/cylc/common/deltas.js

```diff
@@ -79,3 +84,133 @@
  * @property {Array<string>} familyProxies - IDs of family proxies removed
  * @property {Array<string>} jobs - IDs of jobs removed
  */
+
+/**
+ * @typedef {Object} Result
+ * @property {Array<Object>} errors
+ */
+
+/**
+ * @param {DeltasAdded} added
+ * @param {Object.<String, Object>} lookup
+ * @return {Result}
+ */
+function applyDeltasAdded (added, lookup) {
```

Comment:
The code added here was in `workflows/deltas.js`, but was used only by the Tree view. Instead of moving to the tree view, I've moved it here to common since if we ever need to use a delta that creates a lookup, anyone can simply call this function :point_up: 

I **think** I won't need a lookup for GScan, at least for now. Or at least I am avoiding adding one unless really needed. But if I need to add one, then I'll use this function.